projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/answers/Answer.java

```diff
@@ -101,4 +101,8 @@ public void setQuestion(Question question) {
   public void setStatus(AnswerStatus status) {
     _status = status;
   }
+
+  @Override public String toString() {
+    return prettyPrint();
+  }
```

Comment:
this seems an unrelated change. Pull into its own PR with justification?

Reply:
(also, please use the formatter to fix the format of this file.)

Reply:
This changed is added for supporting the toString method for the TestrigQuestion, I can make a separate PR for it~